homeworks/hw1.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+let x = 5;
```

Comment:
Ця і подальші змінні не перезаписується в подальшому виконанні скрипта, тому їх треба об'являти як константи.
```suggestion
const x = 5;
```
і так з наступними

Reply:
Updated.